pkg/activitypub/service/vct/monitor/monitor.go

```diff
@@ -7,6 +7,7 @@ SPDX-License-Identifier: Apache-2.0
 package monitor
```

Comment:
I will rename package to logmonitoring in my next PR. I tried to do it here but it added 500+ lines of code for review because it lost reference to the original file.